### PR TITLE
Added Patch 3.0.1 for HLS plugin

### PR DESCRIPTION
**Pull request by RicardoPoleo**
With this PR we are adding the following fixes:
- Fixed the playlist reload to keep working after interrupting a test and starting a new one.
- Fixed rounding issue when calculating reload time to allow for finer reload times.
- We included a jenv file to fix the java version to be used in dev environments using jenv.

### CODE REVIEW

File: site/dat/repo/various.json [hunk 1]
```diff
@@ -636,6 +636,11 @@
       "com.ubikingenierie.jmeter.plugin.hls.gui.HLSSamplerGUI"
     ],
     "versions": {
+    "6.6.0": {
```
Comment (pmouawad) [inline site/dat/repo/various.json:639]: Can I ask you why the PR included changes on another plugin ?  Please provide a clean PR

File: site/dat/repo/various.json [hunk 2]
```diff
@@ -651,6 +656,13 @@
     "vendor": "adrianmo",
     "markerClass": "io.github.adrianmo.jmeter.backendlistener.azure.AzureBackendClient",
     "versions": {
+      "0.2.1": {
```
Comment (pmouawad) [inline site/dat/repo/various.json:659]: Same here I guess

File: site/dat/repo/various.json (hunk 1 above)
Comment (RicardoPoleo) [reply site/dat/repo/various.json:639]: Hello! I thought it was weird too but, the version that it's now in master contains those lines as well (I just copy-pasted it tbh).
As can be seen here: https://github.com/undera/jmeter-plugins/blob/master/site/dat/repo/various.json

File: site/dat/repo/various.json (hunk 2 above)
Comment (RicardoPoleo) [reply site/dat/repo/various.json:659]: Same goes here. Or is my PR wrong in some way?

Comment (pmouawad) [reply site/dat/repo/various.json:659]: Yes I think so. 
Diff should only concern your parts of code not  parts that are not yours.
I think you sync your fork and make a new PR.

**Pull request merged**